_maps/shuttles/syndicate/syndicate_litieguai.dmm

```diff
@@ -1,1420 +1,1661 @@
 //MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
-"aL" = (
-/obj/machinery/power/smes/engineering{
-	charge = 1e+006
+"am" = (
+/obj/machinery/atmospherics/components/unary/vent_pump/on/layer2{
+	dir = 1
 	},
-/obj/structure/cable{
-	icon_state = "0-2"
+/obj/effect/turf_decal/spline/fancy/opaque/black{
+	dir = 4
+	},
+/obj/effect/turf_decal/spline/fancy/opaque/black{
+	dir = 8
+	},
+/turf/open/floor/plasteel/patterned/brushed,
+/area/ship/medical)
```

Comment:
You might want to consider splitting this up into new areas since the former open-plan medbay has now been broken up into separate rooms. If you do that, just make the whole hallway a single "central hallway" area.